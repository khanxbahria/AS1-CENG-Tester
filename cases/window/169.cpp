#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->changeActiveTabTo(2);
Tab O = Tab("t", "j","o");
wind->moveActiveTabTo(0);
Tab I = Tab("bb", "qqq","z");
wind->isEmpty();
Tab G = Tab("uu", "uu","aa");
wind->isEmpty();
Tab X = Tab("fff", "pp","ii");
wind->newTab(G);
wind->changeActiveTabTo(1);
Tab Y = Tab("sss", "ttt","eee");
wind->changeActiveTabTo(5);
Tab V = Tab("lll", "uu","qq");
wind->moveActiveTabTo(0);
wind->newTab(I);
Tab D = Tab("aa", "o","fff");
wind->changeActiveTabTo(4);
Tab H = Tab("yyy", "f","ggg");
wind->print();
Tab A = Tab("n", "gg","gg");
wind->newTab(A);
Tab E = Tab("hh", "www","rrr");
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(5);
wind->newTab(D);
Tab P = Tab("gg", "ttt","hhh");
wind->moveActiveTabTo(3);
Tab S = Tab("b", "m","qqq");
wind->changeActiveTabTo(5);
wind->isEmpty();
Tab Z = Tab("r", "v","l");
wind->print();
Tab C = Tab("g", "n","vv");
wind->moveActiveTabTo(0);
Tab T = Tab("gg", "cc","h");
wind->isEmpty();
Tab F = Tab("d", "gg","a");
wind->isEmpty();
wind->print();
Tab Q = Tab("v", "u","v");
wind->getActiveTab();
Tab K = Tab("vvv", "qq","bb");
wind->changeActiveTabTo(1);
wind->isEmpty();
wind->changeActiveTabTo(5);
wind->closeTab();
wind->isEmpty();
wind->moveActiveTabTo(5);
Tab N = Tab("rr", "t","s");
wind->getActiveTab();
Tab U = Tab("mm", "vv","mmm");
wind->closeTab();
wind->changeActiveTabTo(2);
wind->moveActiveTabTo(0);
Tab B = Tab("jjj", "bb","nnn");
wind->closeTab();
wind->getActiveTab();
wind->closeTab();
wind->closeTab();
wind->getActiveTab();
wind->moveActiveTabTo(4);
Tab R = Tab("mmm", "iii","a");
wind->print();
Tab L = Tab("jj", "u","yy");
wind->closeTab();
wind->print();
wind->moveActiveTabTo(3);
wind->getActiveTab();
wind->moveActiveTabTo(1);
wind->newTab(B);
wind->closeTab();
wind->isEmpty();
wind->newTab(C);
wind->print();
wind->closeTab();
wind->moveActiveTabTo(0);
wind->getActiveTab();
wind->moveActiveTabTo(3);
wind->newTab(E);
wind->closeTab();
wind->closeTab();
Tab M = Tab("n", "u","kk");
wind->newTab(F);
Tab W = Tab("vv", "nn","sss");
wind->print();
wind->changeActiveTabTo(1);
wind->changeActiveTabTo(4);
wind->newTab(H);
wind->print();
wind->getActiveTab();
wind->closeTab();
wind->print();
wind->moveActiveTabTo(3);
wind->print();
wind->changeActiveTabTo(5);
wind->changeActiveTabTo(1);
wind->isEmpty();
wind->changeActiveTabTo(0);
wind->changeActiveTabTo(1);
wind->closeTab();
wind->changeActiveTabTo(0);
wind->print();
wind->isEmpty();
wind->closeTab();
wind->newTab(K);
wind->changeActiveTabTo(2);
wind->moveActiveTabTo(0);
wind->isEmpty();
Tab J = Tab("qqq", "eee","mm");
wind->getActiveTab();
wind->newTab(J);
wind->changeActiveTabTo(0);
wind->newTab(L);
wind->moveActiveTabTo(3);
wind->closeTab();
wind->isEmpty();
wind->moveActiveTabTo(4);
wind->getActiveTab();
wind->print();
wind->print();
wind->newTab(M);
wind->moveActiveTabTo(5);
wind->changeActiveTabTo(1);
wind->getActiveTab();
wind->print();
wind->isEmpty();
delete wind;
}
	return 0;
}