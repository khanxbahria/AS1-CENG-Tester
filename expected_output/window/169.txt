The active tab is tab 1
uu
uu
aa

qqq
bb
z





The active tab is tab 3
uu
uu
aa

qqq
bb
z





o
aa
fff

gg
n
gg

The active tab is tab 0
o
aa
fff

uu
uu
aa

qqq
bb
z





gg
n
gg

Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0
n
g
vv

The active tab is tab 0
gg
d
a

The active tab is tab 1
gg
d
a

f
yyy
ggg

The active tab is tab 0
gg
d
a

The active tab is tab 0
gg
d
a

Window is empty
The list is empty.
The active tab is tab 1
qq
vvv
bb

eee
qqq
mm

The active tab is tab 1
qq
vvv
bb

eee
qqq
mm

The active tab is tab 1
qq
vvv
bb

eee
qqq
mm

u
n
kk

